packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -407,6 +430,11 @@ contract VotingV2 is
      *       SPAM DELETION FUNCTIONS        *
      ****************************************/
 
+    /**
+     * @notice Declare a specific price requests range to be spam and request it's deletion.
+     * @param spamRequestIndices list of request indices to be declared as spam. Each element is a
+     * pair of uint256s representing the start and end of the range.
+     */
```

Comment:
```suggestion
     * @notice Declare a specific price requests range to be spam and request it's deletion.
     * @dev note that this method should almost never be used. The bond to call this should be set to
     * a very large number (say 10k UMA) as it could be abused if set too low. Function constructs a price
     * request that, if passed, enables pending requests to be diregarded by the contract.
     * @param spamRequestIndices list of request indices to be declared as spam. Each element is a
     * pair of uint256s representing the start and end of the range.
     */
```